apps/catalogue/src/views/HarmonizationRow.vue

```diff
@@ -32,6 +37,10 @@ export default {
     },
     async fetchData() {
       this.resourceMappings = await this.fetchMappings(this.variable);
+      this.resourceStatusMap = this.resources.reduce((statusMap, resource) => {
```

Comment:
suprised to see this inline, I though you did vuex? Otherwise fine, I would also do this in place I suppose

Reply:
vuex manages global ( app global ) state , the harmonization status is kept very local ( per row) due to performance issues ( 'large' amount of data needed to calc cell values combined with grapql overhead). the fetching is done per row ( combined with pageing, 10 rows per batch ) to avoid overloading the server.